src/txn/txn_ckpt.c

```diff
@@ -351,6 +351,8 @@ static int
 __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
 {
 	struct timespec start, stop, verb_timer;
+	struct timespec fsync_start, fsync_stop;
```

Comment:
Generally declarations should be in alphabetical order as well - i.e: this line would be better one line above.